source/documentation/services/SSL-certificate-management.html.md.erb

```diff
@@ -0,0 +1,46 @@
+---
+title: Gandi.net SSL Certificates
+last_reviewed_on: 2021-01-04
+review_in: 3 months
+---
+
+# Gandi.net SSL Certificates
+
+This processes is for any Ministry of Justice users (or suppliers) requiring a Gandi.net SSL Certificate.
+
+There are 2 categories of certificate we use at Ministry of Justice:
+
+* AWS Certificate Manager (ACM)/Let’s Encrypt - automated certificate management for modern cloud native software and infrastructure
+* Gandi.net - where automated certificate management is not possible
+
+Where at all practicable we should utilise [automated certificate management](https://ministryofjustice.github.io/security-guidance/automated-certificate-renewal/#automated-certificate-renewal). The MoJ Hosting Service is looking at strategy to move consumers to modern certificate management solutions.
+
+This process specifically relates to [Standard](https://docs.gandi.net/en/ssl/description/standard.html#standard-certificates) certificates issued through [Gandi.net]
+
+## Requesting a new certificate via Gandi.net
+
+Complete the [MoJ Hosting Service SSL Certificate Request Form](https://docs.google.com/document/d/14XbWoudZd-t4-J3mDBcrAeafAbqxwvdkV-u3Zf8eLOs/edit?usp=sharing) and return it with the Certificate Signing Request (CSR) (and an authority email approval if not an MoJ employee e.g. 3rd party supplier) to [certificates@digital.justice.gov.uk](mailto:certificates@digital.justice.gov.uk).
```

Comment:
Maybe link to docs on how to make a CSR?

Reply:
Good idea. Will add a link.